scss/pages/_civicrm-dashboard.scss

```diff
@@ -1,7 +1,46 @@
+/* stylelint-disable selector-max-id  */
+/* stylelint-disable selector-max-compound-selectors */
+
 .___outside-namespace {
   .page-civicrm-dashboard,
   // targets root `page-civicrm` page and doesn't target child `page-civicrm-event` pages (for example):
   .page-civicrm:not([class*=' page-civicrm-']) {
     @include case-page;
   }
+
+  .civicase__crm-dashboard {
+    &.ui-tabs,
+    .tab-content {
+      background: none;
+    }
+
+    &__tabs.affix {
+      position: fixed;
+      top: 0;
+      width: 100%;
+      z-index: 9999;
+    }
+  }
+
+  #crm-container .civicase__crm-dashboard__tabs {
```

Comment:
This should be a new component. 
Lets not call it `civicase__crm-dashboard__tabs` as it can be used for other tabs outside of shoreditch too.
We can call it `civicase__tabs`, also add some documentation in its partial, about why it needed to be defined.

Reply:
fixed.